semeio/workflows/localisation/local_script_lib.py

```diff
@@ -744,6 +777,155 @@ def add_ministeps(
+            raise ValueError(
+                f"For ministep: {ministep.name()} there does not exist "
+                f"any parameter group : {param_group_name}"
+            )
+        param_group = ministep[param_group_name]
+        node_names = list(param_group.keys())
+        for node_name in node_names:
+            node = ert_ensemble_config.getNode(node_name)
+            impl_type = node.getImplementationType()
+            active_list_obj = param_group.getActiveList(node_name)
+            if node_name not in param_dict:
+                raise ValueError(
+                    f"Ministep {ministep.name()} with parameter group "
+                    f"{param_group_name} has node name {node_name} "
+                    "that is not specified."
+                )
+            user_spec_param_list = param_dict[node_name]
+
+            # Check only cases with partly active set of parameter
+            if active_list_obj.getMode() == ActiveMode.PARTLY_ACTIVE:
+                if impl_type == ErtImplType.GEN_KW:
+                    spec_index_list = []
+                    for nr, user_param_name in enumerate(user_spec_param_list):
+                        spec_index_list.append(
+                            active_index_for_parameter(
+                                node_name, user_param_name, ert_param_dict
+                            )
+                        )
+
+                elif impl_type == ErtImplType.GEN_DATA:
+                    spec_index_list = [
+                        int(user_spec_param_list[i])
+                        for i in range(len(user_spec_param_list))
+                    ]
+
+                active_index_list = active_list_obj.get_active_index_list()
+                spec_index_list.sort()
+                active_index_list.sort()
+                if len(spec_index_list) != len(active_index_list):
+                    raise ValueError(
+                        f"For ministep: {ministep.name()} the number of "
+                        "active parameters are: "
+                        f"{len(active_index_list)} \n"
+                        "while the specified number of active parameters "
+                        f"are: {len(spec_index_list)}"
+                    )
+                err = False
+                for nr, index in enumerate(active_index_list):
+                    if index != int(spec_index_list[nr]):
+                        err = True
+                if err:
+                    raise ValueError(
+                        f" For ministep: {ministep.name()} and "
+                        f"parameter node: {node_name}:\n"
+                        "Mismatch between specified active parameters "
+                        f"and active parameters in the ministep.\n"
+                        f"Specified: {spec_index_list}\n"
+                        f"In ministep: {active_index_list}\n"
+                    )
+
+
+def verify_ministep_active_obs(corr_spec_list, ert):
```

Comment:
Same comment as `verify_ministep_active_param`

Reply:
Is moved to test_integration.py